src/static/styles/partials/_request-progress.css

```
@@ -0,0 +1,334 @@
+    --compat-placeholder: 1;
+}
+.tooltip{visibility: hidden;}
+.tooltip-text{visibility: visible;}
+.live-preview-overlay{backdrop-filter: blur(4px); background: rgba(0,0,0,0.4);}
+@keyframes slide-in-from-right{from{opacity:0;}to{opacity:1;}}
+@keyframes slide-in-from-bottom{from{opacity:0;}to{opacity:1;}}
+/* @keyframes slideInFromRight */
+/* @keyframes slideInFromBottom */
+/* Enhanced Typography Hierarchy */
+/* Mobile adjustments for live preview */
+.live-preview-overlay{bottom: 20px;}
```

Stylelint (17.7.0)

[error] Duplicate selector ".live-preview-overlay", first used at line 271 (no-duplicate-selectors)

(no-duplicate-selectors)